android/java/res/layout/activity_brave_leo_plans.xml

```diff
@@ -9,86 +9,359 @@
+                                    android:padding="8dp"
+                                    app:srcCompat="@drawable/ic_leo_chat"
+                                    android:contentDescription="@null"/>
+
+                            <TextView
+                                    android:id="@+id/tv_chat"
+                                    android:layout_width="0dp"
+                                    android:layout_height="wrap_content"
+                                    android:text="@string/leo_tip_chat_title"
+                                    android:textSize="16sp"
+                                    android:textColor="@android:color/white"
+                                    android:layout_marginStart="12dp"
+                                    android:layout_marginTop="8dp"
+                                    app:layout_constraintEnd_toEndOf="parent"
+                                    app:layout_constraintStart_toEndOf="@id/iv_chat"
+                                    app:layout_constraintTop_toTopOf="parent"/>
+
+                            <TextView
+                                    android:layout_width="0dp"
+                                    android:layout_height="wrap_content"
+                                    android:text="@string/leo_tip_chat_description"
+                                    android:textSize="12sp"
+                                    android:textColor="@color/brave_leo_plans_tips_desc"
+                                    android:layout_marginStart="12dp"
+                                    app:layout_constraintEnd_toEndOf="parent"
+                                    app:layout_constraintStart_toEndOf="@id/iv_chat"
+                                    app:layout_constraintTop_toBottomOf="@id/tv_chat"/>
+
+                        </androidx.constraintlayout.widget.ConstraintLayout>
+
+                    </LinearLayout>
+
+                    <LinearLayout
+                        android:id="@+id/monthly_selector_layout"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginBottom="10dp"
+                        android:layout_marginHorizontal="24dp"
+                        android:layout_marginTop="14dp"
+                        android:baselineAligned="false"
+                        android:padding="16dp"
+                        android:background="@drawable/leo_plan_monthly_bg"
+                        android:orientation="horizontal">
+
+                        <LinearLayout
+                            android:layout_width="0dp"
+                            android:layout_height="wrap_content"
+                            android:layout_weight="1"
+                            android:layout_gravity="center"
+                            android:orientation="vertical">
+
+                            <TextView
```

Comment:
nit : Why do we need Linearlayout as parent here ?

Reply:
yeah you are right, not needed. I removed it.